infrastructure/proto/log_replication_metadata.proto

```diff
@@ -30,12 +35,42 @@ message LogReplicationMetadataVal {
   string val = 1;
 }
 
+/*--------END---------*/
+
+/*------- Log Replication Metadata Table V2 Schema ----------*/
+
+message ReplicationMetadata {
+  uint64 topologyConfigId = 1;
+  string version = 2;
+  int64 lastSnapshotStarted = 3;
+  int64 lastSnapshotTransferredSeqNumber = 4;
+  int64 lastSnapshotTransferred = 5;
+  int64 lastSnapshotApplied = 6;
+  int64 lastLogEntryApplied = 7;
+  org.corfudb.runtime.UuidMsg currentSnapshotCycleId = 8;
+  int64 currentCycleMinShadowStreamTs = 9;
+  int64 remainingReplicationPercent = 10;
```

Comment:
How will sink be populating remainingReplicationPercent?

Reply:
yes.. not sure why that and 'dataConsistent' fields are present...

Reply:
dataConsistentOnStandby on sink makes sense since sink knows when snapshot_sync is completed.
But not sure about remainingReplicationPercent

Reply:
yes, but that field was always read from the status.

But good catch indeed.  Removed.